pkg/jobrunaggregator/jobrunhistoricaldataanalyzer/analyzer.go

```diff
@@ -11,6 +11,8 @@ import (
 	"github.com/openshift/ci-tools/pkg/jobrunaggregator/jobrunaggregatorlib"
 )
 
+const releaseTransitionDataCountThreshold = 60
```

Comment:
Am I correct that this is intended to require at least 60% as many results as we had before? Suggest making this 0.6, maybe renaming to releaseTransitionDataMinThresholdRatio, and godoc to explain that we require at least this many results before we'll flip the data file over to the latest release, otherwise we keep using the old.